### ISSUE TITLE
Maybe there is a bug in this line

### ISSUE THREAD

**Issue by samotiian** [yuval-alaluf/restyle-encoder#40]
https://github.com/yuval-alaluf/restyle-encoder/blob/7306b2be8ba6a7b498cf2b3ed4c60330b8bc7c27/training/coach_restyle_e4e.py#L319

Shouldn't it be  'loss += ' instead of 'loss = '?

**Comment by yuval-alaluf** [comment]
Thanks for pointing it out. I just pushed a fixed. It seems like this was accidentally changed when I first published the code. The pre-trained models for e4e should still be good to go.
